src/admin/e2e/authentication.js

```diff
@@ -0,0 +1,15 @@
+/* eslint func-names: off */
+import { until, By } from 'selenium-webdriver';
+import { describe, it } from 'selenium-webdriver/testing';
```

Comment:
Not sure it's needed now that we have async in mocha.
http://seleniumhq.github.io/selenium/docs/api/javascript/module/selenium-webdriver/testing/index.html